app/queue.py

```diff
@@ -22,6 +22,9 @@ class Buffer(Enum):
     def name(self, suffix=None):
         return f"{self.value}:{suffix}" if suffix else self.value
 
+    def get_inflight_name(self, receipt: UUID = uuid4(), suffix: str = "") -> str:
+        return f"{Buffer.IN_FLIGHT.value}:{str(suffix)}:{str(receipt)}"
```

Comment:
@iokpala There is a chance there will be two consecutive `:` if the `suffix` parameter is empty. Can we please make the parameter default to `None` instead and have a ternary operator (similar to line 23 for the `name` function) so that it removes the extra `:` if suffix is not defined?

Reply:
I noticed that as well..  Will fix it